### PR TITLE
Fix SDK repo links in "Available SDKs" & SDK tutorials

### PR DESCRIPTION
**Pull request by chenette**
Several SDKs moved into separate repositories (Go, Java, JS, and C++)

Signed-off-by: Anne Chenette <chenette@bitwise.io>

### CODE REVIEW

File: docs/source/_templates/sdk_overview_tutorial.rst [hunk 1]
```diff
@@ -53,17 +53,42 @@ to create transactions and submit them as :term:`Sawtooth batches<Batch>`.
...
+{% elif language == 'JavaScript' %}
+
+   For a full JavaScript implementation of the XO transaction family, see
+   `https://github.com/hyperledger/sawtooth-sdk-javascript/tree/master/examples/xo
+   <https://github.com/hyperledger/sawtooth-sdk-javascript/tree/master/examples/xo>`_.
+
+{% else %}
+
+   For a full Python implementation of the XO transaction family, see
+   ``/{project}/sawtooth-core/sdk/examples/xo_{{ lowercase_lang }}/``.
```
Comment (aludvik) [inline docs/source/_templates/sdk_overview_tutorial.rst]: This link does not expand in the generated HTML

File: docs/source/_templates/sdk_overview_tutorial.rst [hunk 2]
```diff
@@ -53,17 +53,42 @@ to create transactions and submit them as :term:`Sawtooth batches<Batch>`.
...
 
-   * For a full implementation of the tic-tac-toe transaction family, see
-     ``/project/sawtooth-core/sdk/examples/xo_{{ lowercase_lang }}/``.
 
-   * For full implementations in other languages, see
-     ``https://github.com/hyperledger/sawtooth-core/tree/master/sdk/examples``.
+{% if language == 'Rust' %}
+
+   For a full Rust implementation of the XO transaction family, see
+   ``/{project}/sawtooth-core/sdk/examples/xo_{{ lowercase_lang }}/``.
```
Comment (aludvik) [inline docs/source/_templates/sdk_overview_tutorial.rst]: This link does not expand in the generated HTML

File: docs/source/_templates/sdk_overview_tutorial.rst (hunk 1 above)
Comment (chenette) [reply docs/source/_templates/sdk_overview_tutorial.rst]: This isn't a link - it's just showing the path to the location in the user's local fork of the repo.  The only change I made was to put curly brackets around {project} to make it clear(er) that you don't type "project" when changing to that directory.

File: docs/source/_templates/sdk_overview_tutorial.rst (hunk 2 above)
Comment (chenette) [reply docs/source/_templates/sdk_overview_tutorial.rst]: (ditto)

**Pull request merged**